vespalib/src/tests/stllike/string_test.cpp

```diff
@@ -449,4 +449,10 @@ TEST("starts_with has expected semantics for std::string_view") {
     EXPECT_FALSE(ar.starts_with("oobar"));
 }
 
+TEST("test allowed nullptr construction") {
+    EXPECT_TRUE(vespalib::string(nullptr, 0).empty());
```

Comment:
Consider adding a comment about this being legal but not legitimate cases.

Reply:
Done